<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainwindow</class>
 <widget class="QMainWindow" name="Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loginwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_lineedit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>114</width>
      <height>19</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_lineedit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_Button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Message:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="message_Button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Say it!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="message_textEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>601</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
